EasyPost.Tests.Net/BatchTest.cs

```diff
@@ -1,232 +1,172 @@
                     }
-                },
-                {
-                    "to_address", toAddress
-                },
-                {
-                    "from_address", fromAddress
                 }
-            };
-            batchShipmentParameters = new Dictionary<string, object>
-            {
-                {
-                    "parcel", new Dictionary<string, object>
-                    {
-                        {
-                            "length", 8
-                        },
-                        {
-                            "width", 6
-                        },
-                        {
-                            "height", 5
-                        },
-                        {
-                            "weight", 10
-                        }
-                    }
-                },
-                {
-                    "to_address", toAddress
-                },
-                {
-                    "from_address", fromAddress
-                },
-                {
-                    "carrier", "USPS"
-                },
-                {
-                    "service", "Priority"
-                }
-            };
+            });
         }
 
         [TestMethod]
-        public void TestAddRemoveShipments()
+        public void TestCreate()
         {
-            Batch batch = Batch.Create();
-            Shipment shipment = Shipment.Create(shipmentParameters);
-            Shipment otherShipment = Shipment.Create(shipmentParameters);
+            VCR.Replay("create");
 
-            while (batch.state != "created")
-            {
-                batch = Batch.Retrieve(batch.id);
-            }
+            Batch batch = CreateBasicBatch();
 
-            batch.AddShipments(new List<Shipment>
+            Assert.IsInstanceOfType(batch, typeof(Batch));
+            Assert.IsTrue(batch.id.StartsWith("batch_"));
+            Assert.IsNotNull(batch.shipments);
+        }
+
+        [TestMethod]
+        public void TestRetrieve()
+        {
+            VCR.Replay("retrieve");
+
+            // use "testCreate"
+            Batch batch = CreateBasicBatch();
+
+            Batch retrievedBatch = Batch.Retrieve(batch.id);
+
+            Assert.IsInstanceOfType(retrievedBatch, typeof(Batch));
+            Assert.AreEqual(batch.id, retrievedBatch.id);
+        }
+
+        [TestMethod]
+        public void TestAll()
+        {
+            VCR.Replay("all");
+
+            BatchCollection batchCollection = Batch.All(new Dictionary<string, object>
             {
-                shipment,
-                otherShipment
+                {
+                    "page_size", Fixture.PageSize
+                }
             });
 
-            while (batch.shipments == null)
+            List<Batch> batches = batchCollection.batches;
+
+            Assert.IsTrue(batches.Count <= Fixture.PageSize);
+            Assert.IsNotNull(batchCollection.has_more);
+            foreach (var item in batches)
```

Comment:
I would rename the `item` variable to singular of the looping object name. This applies to the rest of the files.